client/pages/signup.html

```diff
@@ -28,12 +28,41 @@ <h1> Sign Up </h1>
 		</form>
 	
 	</div>
+
+	<nav class="navbar navbar-default navbar-expand navbar-dark bg-dark">
+		<a href="#" class="navbar-brand">Torchbearer</a>
+		<button class="navbar-toggler" data-toggle="collapse" data-target="#navbarMenu">
+			<span class="navbar-toggler-icon"></span>
+	</button>
+	<div class="collapse navbar-collapse" id="navbarMenu">
+		<ul class="navbar-nav">
+			<li class="nav-item">
+				<a href="#" class="nav-link">Home</a>
+			</li>
+			<li class="nav-item">
+				<a href="#" class="nav-link">Board</a>
+			</li>
+			<li class="nav-item">
+				<a href="#" class="nav-link">Tasks</a>
+			</li>
+
+		</ul>
+
+	</div>
+
+
+</nav> 
 	
 	<!-- ---------------------------------------- Bootstrap ---------------------------------------- -->
 	<script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
 	<script src = "https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js"></script>
+
+	<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+	
 	<!-- -------------------------------------- Local Scripts -------------------------------------- -->
-	<script src = "../scripts/signup.js"> </script>
+	<script src = "/scripts/signup.js"> </script>
```

Comment:
While this will work using live server this won't when we eventually deploy our application. This means go to the root repository and then go into the scripts folder. The root repository can change based on how it's hosted. The correct way to do this is `../scripts/signup.js` which means go back 1 folder and then go into the scripts folder.